docs/modders/Map_Objects/Dwelling.md

```diff
@@ -22,7 +22,10 @@
 	"guards" : true,
 	"guards" : [
 		{ "amount" : 12, "type" : "earthElemental" }
-	]
+	],
+
+	/// Optional image showed on kingdom overview (animation; only frame 0 displayed)
```

Comment:
Perhaps remove "optional"part, at least from docs?

This image is required for dwelling to function correctly after all.

Would be great to also enforce it via schema, but looks like dwellings don't have one at the moment & that would cause validation failure in multiple mods, so probably not for 1.7